doc/building.md

```diff
@@ -1261,59 +1334,61 @@ Architectures that are known to successfully cross-compile like this are:
 | sh4          | sid          | sh4           | sh4-linux-gnu            | zero                      |
 | riscv64      | sid          | riscv64       | riscv64-linux-gnu        | (all)                     |
 
-### Building for ARM/aarch64
+### Considerations for specific targets
+
+#### Building for ARM/aarch64
```

Comment:
I don't think we need to or recommend setting ABI for aarch64.

Reply:
Yep. I'll fix the heading.